torch/onnx/symbolic_opset9.py

```diff
@@ -1493,6 +1493,17 @@ def scalar_tensor(g, scalar, dtype, *options):
     scalar = g.op("Cast", scalar, to_i=sym_help.scalar_type_to_onnx[dtype])
     return scalar
 
+def tensor(g, data, dtype=None, device=None, requires_grad=False):
+    value = sym_help._maybe_get_const(data, 't')
+    if dtype is None:
+        dtype = 6  # float
+    else:
+        if sym_help._is_packed_list(data):
```

Comment:
Does the test only cover this case? I suggest having tests for all cases in the if-else branch

Reply:
Added test for if-else branch